### PR TITLE
chore: Portuguese translations for the labels taxonomy

### PR DESCRIPTION
**Pull request by laralem**
All Portuguese translations added and a few fixed, with the following exceptions:
-Added in Spanish: "Alto Contenido en Fibra" (checked in a cereals box)
-1 missing space and 1 double space in 2 french translations

At the end of document I've added labels for Portuguese protected products, not all of them but the most important (wines and meat). They are too much or there is no problem with that? Should I add all of them? (almost 100 protected products left).

### CODE REVIEW

**Comment by teolemon** [comment]
Errors in the labels taxonomy definition:
ERROR - pt:galinhas-ao-ar-livre already is a synonym of pt:galinhas-criadas-em-liberdade for entry en:free-range-chicken - pt:aves-criadas-em-liberdade cannot be mapped to entry en:free-range-poultry
ERROR - pt:indicacao-geografica-protegida already is a synonym of pt:igp for entry en:pgi - pt:indicacao-geografica-protegida cannot be mapped to entry pt:indicacao-geografica-protegida
ERROR - pt:igp already is a synonym of pt:igp for entry en:pgi - pt:indicacao-geografica-protegida cannot be mapped to entry pt:indicacao-geografica-protegida
Errors in the labels taxonomy definition at lib/ProductOpener/Tags.pm line 994.

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -19054,3 +19309,148 @@ xx:Recicla amarillo
 
 es:Recicla azul
 xx:Recicla azul
+
+en:Portuguese protected product
+pt:Produto protegido de Portugal
+
+pt:Especialidade Tradicional Garantida, ETG, E.T.G.
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:19316]: this should be removed as it's already here:

en:TSG, t.S.g., Traditional Speciality Guaranteed
bg:храна с традиционно специфичен характер, ХТСХ
cs:zaručená tradiční specialita, ZTS
da:garanteret traditionel specialitet, GTS
de:gtS, g.t.S., garantiert traditionelle Spezialität
el:εγγυημένο παραδοσιακό ιδιότυπο προϊόν, ΕΠΙΠ
es:Especialidad tradicional garantizada, ETG, E.T.G.
et:garanteeritud traditsiooniline eritunnus, GTE
fi:APT, A.P.T, aito perinteinen tuote
fr:STG, S.t.g., Spécialité traditionnelle garantie
ga:speisialtacht thraidisiúnta ráthaithe, STR
hu:Hagyományos különleges termék, HKT
it:specialità tradizionale garantita, STG
lt:garantuotas tradicinis gaminys, GTG
lv:garantēta tradicionālā īpatnība, GTI
mt:speċjalità tradizzjonali garantita, STG
nl:gegarandeerde traditionele specialiteit, GTS
pl:gwarantowana tradycyjna specjalność, GTS
pt:especialidade tradicional garantida, ETG

Same for pt:Especialidade Tradicional Garantida

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -242,14 +263,14 @@ en:Free range chicken
 de:Freilandhühnchen
 fi:ulkokana, vapaana elänyt kana
 hu:Szabad tartású csirke, Szabad tartású tyúk
-pt:Galinhas criadas em liberdade
+pt:Galinhas criadas em liberdade, galinhas em liberdade, galinhas livres de gaiolas, galinhas criadas ao ar livre, galinhas ao ar livre
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:266]: galinhas ao ar livre can be a  synonym for en:free range chicken or en:free range poultry, but not both.

**Comment by benbenben2** [comment]
It was pending during quite some time but finally, your contribution has been merged @laralem 

We will close this PR because it has been replaced by https://github.com/openfoodfacts/openfoodfacts-server/pull/8356

**Pull request closed without merging**